app/views/councillor_contributions/new.html.haml

```diff
@@ -0,0 +1,17 @@
+%h1 Add a new councillor for #{@authority.full_name}
+%h2 The councillor(s) you submitted:
```

Comment:
This heading the `<ul>` of submitted councillors shouldn't render when the person hasn't submitted any councillors yet.